File: app/src/main/java/black/bracken/shotsorter/util/IntUtil.java
package black.bracken.shotsorter.util;

/**
 * @author BlackBracken
 */
public final class IntUtil {

    private IntUtil() {
    }

    public static int parseIntOrDefault(String parsed, int defaultValue) {
        try {
            return Integer.parseInt(parsed);
        } catch (NumberFormatException ex) {
            return defaultValue;
        }
    }

}

File: app/src/main/java/black/bracken/shotsorter/util/IntUtil.kt
package black.bracken.shotsorter.util

/**
 * @author BlackBracken
 */
object IntUtil {

    fun parseIntOrDefault(parsed: String, defaultValue: Int): Int {
        try {
            return Integer.parseInt(parsed)
        } catch (ex: NumberFormatException) {
            return defaultValue
        }

    }

}
